### ISSUE TITLE
Raspberry Pi - Analog Audio Output?

### ISSUE THREAD

**Issue by CemTezcan** [randyrossi/bmc64#132]
I don't know if it's possible or this may me out of the scope but Pizeros have an analog video pins on the PCB but there is no audio out. To use BMC 64 Lite with real CRT screens with audio there needs to be an analog audio out.

I checked that there is a tutorial that makes this happen.
http://shallowsky.com/blog/hardware/pi-zero-audio.html

It mentions about adding a command to config file.
dtoverlay=pwm-2chan,pin=18,func=2,pin2=13,func2=4

Can this be applied to BMC 64 lite to override pin config?

**Comment by randyrossi** [comment]
Yes, this is doable.  PWM audio is what's needed.  It's supported by circle but I'd have to make some code changes.  I'm taking a bit of a break from bmc64 stuff for a while but I'll get to this eventually.

**Comment by SvOlli** [comment]
You might want to add a bit of hardware as well. Adafruit has a nice document on how to add audio to the Pi Zero: https://learn.adafruit.com/introducing-the-raspberry-pi-zero/audio-outputs . Hope this helps.

**Comment by maitrezoid** [comment]
Hi. Is there something new with analog output on pi zero ? I can't get it working.

**Comment by maitrezoid** [comment]
Hi. I saw audio can be set to PWM Gpio 12,3,18 or 19, but those Gpio are already used for real keyboard.
